drivers/SmartThings/philips-hue/src/hue/api.lua

```diff
@@ -115,17 +115,16 @@ function PhilipsHueApi.new_bridge_manager(base_url, api_key, socket_builder)
       local msg, err = control_rx:receive()
       if err then
         if err ~= "timeout" then
-          log.error_with({ hub_logs = true }, "[PhilipsHueApi] Error receiving on control channel for REST API thread", err)
+          log.error("[PhilipsHueApi] Error receiving on control channel for REST API thread", err)
         else
-          -- TODO convert this to TRACE logs when debugging is over: dougstephen@smartthings.com
-          log.info_with({ hub_logs = true }, "Timeout on Hue API Control Channel, continuing")
+          log.trace("Timeout on Hue API Control Channel, continuing")
```

Comment:
This log has ended up being a nice little heartbeat indicator but do we feel that there are problems with printing this to the live logs every 30 seconds?